components/grid/demo/playground.md

```diff
@@ -89,11 +89,11 @@ ReactDOM.render(<App />, mountNode);
 ````
 
 ````css
-#components-grid-demo-playground [class^="ant-col-"] {
+#components-grid-demo-playground [class~="ant-col"] {
```

Comment:
`class*="ant-col-"`

Reply:
`[class~="ant-col"]`: ``~classValue.split(' ').indexOf('ant-col')``
`[class*="ant-col-"]`: `~classValue.indexOf('ant-col-')`
Are there some advantages to using `*=` ?

Reply:
I missed that:
```diff
- ant-col-
+ ant-col
```

Your solution is better.